algoliasearch-core/src/test/java/com/algolia/search/JacksonParserTest.java

```diff
@@ -182,7 +182,7 @@ void testLegacyFiltersFormat(String input) throws IOException {
     // Testing "one array" legacy filters => should be converted to "ORED" nested filters
     // [["color:green","color:yellow"]]
```

Comment:
IIUC that the expected result:

```suggestion
    // [["color:green"],["color:yellow"]]
```